PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/AzurePlugin.java

```diff
@@ -133,52 +110,7 @@ public void initComponent() {
         }
     }
 
-    private void initializeTelemetry() throws Exception {
```

Comment:
Don't move the `initializeTelemetry()` out of the base class please, which should be enabled in every IDEs.

Reply:
Ok, will return